teradatalabs/mapr52-base/Dockerfile

```diff
@@ -0,0 +1,69 @@
+# Copyright 2017 Teradata
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM teradatalabs/centos6-java8-oracle
+MAINTAINER Teradata Docker Team <docker@teradata.com>
+
+# ADD REPO FOR MAPR
+ADD files/maprtech.repo /etc/yum.repos.d/maprtech.repo
+COPY files/id_rsa.pub /root/
+RUN yum update -y \
+# ... GET MapRGPG KEY
+  && rpm --import http://package.mapr.com/releases/pub/maprgpg.key \
+
+# INSTALL UTILITY SOFTWARE
+  && yum install -y iputils vim openssh-server openssh-clients sudo lsof \
+# CONFIGURE SSH
+  && chkconfig sshd on \
+  && grep -rl '#Port 22'  /etc/ssh/sshd_config | xargs sed -i 's/#Port 22/Port 22/g' \
```

Comment:
Its not working without that , other containers will all have same problems when some one will ssh hadoop-master from outside.